scripts/climate

```diff
@@ -49,7 +49,7 @@ my $base = abs_path(getcwd);
 
 # The threshold is the percentage in which we consider that the given package
 # is covered enough by tests. It defaults to 90.0%
-my $threshold = 90.0;
+my $threshold = 75.0;
```

Comment:
I'd prefer if we added tests rather than dropping the minimum test coverage.

Reply:
i had a chat with @mssola about that, and he said, this should do it for now..